pycqed/instrument_drivers/physical_instruments/ZurichInstruments/UHFQuantumController.py

```diff
@@ -759,16 +759,19 @@ def awg_update_waveform(self, index, data) -> None:
     ##########################################################################
 
     def acquisition(self, samples=100, averages=1, acquisition_time=0.010, timeout=10,
-                    channels=(0, 1), mode='rl') -> None:  # FIXME: wrong return type
+                    channels=(0, 1), mode='rl', poll=True):
```

Comment:
Again, we should not allow this PR to include any conflicting changes with the UHFQC driver!